### ISSUE TITLE
ngfSrc directive injects $parse service twice

### ISSUE THREAD

**Issue by stefanFruuu** [danialfarid/ng-file-upload#734]
```javascript
ngFileUpload.directive("ngfSrc", [ "$parse", "$timeout", "$parse", function($parse, $timeout, $parse)
```

**Comment by Ferdaszewski** [comment]
This is causing a syntax error when in strict mode: `Argument name clash in strict mode`

**Comment by danialfarid** [comment]
fixed in version 4.2.1
